rasa_core/channels/direct.py

```diff
@@ -16,4 +16,7 @@ def latest_output(self):
             return None
 
     def send_text_message(self, recipient_id, message):
-        self.messages.append((recipient_id, message))
+        self.messages.append({"recipient_id": recipient_id, "text": message})
```

Comment:
@tmbo before I go update a bunch of tests - does this make sense to change? idea is for collectingoutputchannel to collect full messages with a `data` attribute, not just strings

Reply:
sounds good :+1